core/server/src/main/java/alluxio/proxy/FileSystemClientRestServiceHandler.java

```diff
@@ -59,24 +58,34 @@
 @NotThreadSafe
 @Path(FileSystemClientRestServiceHandler.SERVICE_PREFIX)
 @Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
 public final class FileSystemClientRestServiceHandler {
-  public static final String SERVICE_PREFIX = "file";
-  public static final String SERVICE_NAME = "service_name";
-  public static final String SERVICE_VERSION = "service_version";
-  public static final String CREATE_DIRECTORY = "create_directory";
+  public static final String SERVICE_PREFIX = "alluxio";
```

Comment:
not really, but if we don't then I would need to split up the implementation of the API into two classes, which I am happy to do